src/mscorlib/shared/System/Char.cs

```diff
@@ -26,7 +26,7 @@ public struct Char : IComparable, IComparable<Char>, IEquatable<Char>, IConverti
         //
         // Member Variables
         //
-        private char _value;
+        private char m_value; // Do not rename (binary serialization)
```

Comment:
Could you please share details about the issues that you are hitting that are "fixed" by this?